tls/s2n_connection.c

```diff
@@ -853,11 +853,17 @@ int s2n_connection_use_corked_io(struct s2n_connection *conn)
 
 uint64_t s2n_connection_get_wire_bytes_in(struct s2n_connection *conn)
 {
+    if (conn->ktls_recv_enabled) {
+        return 0;
+    }
```

Comment:
We should make sure to document this on the method

Reply:
Definitely. But I don't want to mention ktls on the public documentation just yet.